readthedocs/rtd_tests/tests/test_domains.py

```diff
@@ -99,6 +99,15 @@ def test_public_domain_not_allowed(self):
             f'{settings.PUBLIC_DOMAIN} is not a valid domain.'
         )
 
+    def test_domain_with_path(self):
+        form = DomainForm(
+            {'domain': 'domain.com/foo/bar'},
+            project=self.project,
+        )
+        self.assertTrue(form.is_valid())
+        domain = form.save()
+        self.assertEqual(domain.domain, 'domain.com')
```

Comment:
It might be worth adding some test cases with more garbage data as well too

Reply:
I have added some, but mostly aren't being caught by https://github.com/readthedocs/readthedocs.org/pull/7502/files#r493113673